src/htrun/README.md

```diff
@@ -261,9 +261,9 @@ Key-value protocol was developed and is used to provide communication layer betw
 
 # Greentea client API
```

Comment:
In my opinion, anything below can be split into a separate file (e.g. programming guide). Also, we could consider moving docs into a `docs/` directory of this repository. This might be outside the scope of the PR and can be done in a follow-up PR.